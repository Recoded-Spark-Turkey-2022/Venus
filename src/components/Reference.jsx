import React from "react";
import { Swiper, SwiperSlide } from "swiper/react";
import {Pagination, Autoplay } from "swiper";
import 'swiper/swiper-bundle.css';

import '../index.css'
import worldMap from '../assets/world_map_ 1.svg'


function Reference (){


    return(
        <div className="flex flex-col md:flex-row justify-around items-center py-12 ">
            <div className="flex justify-center items-center ">
                <img className="w-2/3 md:w-full" src={worldMap} alt="World Map"  />
            </div>
            <div className="flex justify-end w-full md:w-1/2 ">
            <Swiper
        autoplay={{
          delay: 2500,
          disableOnInteraction: false,
        }}
    
          pagination={{
            clickable: true,  dynamicBullets: true 
          }}
          
          modules={[Pagination, Autoplay]}
          className="mySwiper">
            <SwiperSlide>
                <div className="w-1/2 reference-div text-center sm:text-left">
                    <h3 className="pt-2">Two lines of a short testimonial from someone want to say something, and can
                     say more to explain.</h3>
                    <p className="text-darkGrey pt-2">Louis Li</p>
                    <p className="text-darkGrey text-sm">Trainer at Recoded</p>
                </div>
                </SwiperSlide>
                <SwiperSlide>
                <div className="w-1/2 reference-div text-center sm:text-left">
                    <h3 className="pt-2">Two lines of a short testimonial from someone want to say something, and can
                     say more to explain.</h3>
                    <p className="text-darkGrey pt-2">Louis Li</p>
                    <p className="text-darkGrey text-sm">Trainer at Recoded</p>
                </div>
                </SwiperSlide>
                <SwiperSlide>
                <div className="w-1/2 reference-div text-center sm:text-left">
                    <h3 className="pt-2">Two lines of a short testimonial from someone want to say something, and can
                     say more to explain.</h3>
                    <p className="text-darkGrey pt-2">Louis Li</p>
                    <p className="text-darkGrey text-sm">Trainer at Recoded</p>
                </div>
                </SwiperSlide>
                </Swiper>
            </div>

        </div>
    )
}

export default Reference;