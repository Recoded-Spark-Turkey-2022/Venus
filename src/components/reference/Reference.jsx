import React from 'react';
import { Swiper, SwiperSlide } from 'swiper/react';
import { Pagination, Autoplay } from 'swiper';
import 'swiper/swiper-bundle.css';
import { motion } from 'framer-motion';
import { useTranslation } from 'react-i18next';

<<<<<<<< HEAD:src/components/Reference/Reference.jsx
import './Reference.css';
========

import '../../index.css';

import './Reference.css';

>>>>>>>> c91c9e5b9e8c0f66102acaa3fab218683e6f7006:src/components/reference/Reference.jsx
import worldMap from '../../assets/world_map_ 1.svg';

function Reference() {
  const { t }= useTranslation();
  return (
    <motion.div
      initial={{ opacity: 0, scale: 0.5 }}
      animate={{ opacity: 1, scale: 1 }}
      transition={{ duration: 0.7 }}
      className="flex flex-col md:flex-row justify-around items-center py-12 "
    >
      <div className="flex justify-center items-center ">
        <img className="w-2/3 md:w-full" src={worldMap} alt="World Map " />
      </div>
      <div className="flex justify-end w-full md:w-1/2 ">
        <Swiper
          autoplay={{
            delay: 2500,
            disableOnInteraction: false,
          }}
          pagination={{
            clickable: true,
            dynamicBullets: true,
          }}
          modules={[Pagination, Autoplay]}
          className="mySwiper pt-[100px]"
        >
          <SwiperSlide>
            <div className="w-1/2 reference-div text-center sm:text-left pb-[65px]">
              <h3 className="pt-2">
              {t("Home.r1")}
              </h3>
              <p className="text-[#b7b7b7] pt-2">{t("Home.r1-h")}</p>
              <p className="text-[#b7b7b7] text-sm">
              {t("Home.r1-p")}
              </p>
            </div>
          </SwiperSlide>
          <SwiperSlide>
            <div className="w-1/2 reference-div text-center sm:text-left pb-[65px]">
              <h3 className="pt-2">
              {t("Home.r2")}
              </h3>
              <p className="text-[#b7b7b7] pt-2">{t("Home.r2-h")}</p>
              <p className="text-[#b7b7b7] text-sm">
              {t("Home.r2-p")}
              </p>
            </div>
          </SwiperSlide>
          <SwiperSlide>
            <div className="w-1/2 reference-div text-center sm:text-left pb-[65px]">
              <h3 className="pt-2">
              {t("Home.r3")}
              </h3>
              <p className="text-[#b7b7b7] pt-2">{t("Home.r3-h")}</p>
              <p className="text-[#b7b7b7] text-sm">
              {t("Home.r3-p")}
              </p>
            </div>
          </SwiperSlide>
        </Swiper>
      </div>
    </motion.div>
  );
}

export default Reference;
