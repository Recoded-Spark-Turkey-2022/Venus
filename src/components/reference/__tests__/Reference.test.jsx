import React from 'react';
import renderer from 'react-test-renderer';
import Reference from '../Reference';

jest.mock('swiper/react', () => ({
  Swiper: 'Swiper',
  SwiperSlide: 'SwiperSlide',
}));

jest.mock('swiper', () => ({
  Pagination: 'Pagination',
}));

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: (str) => str
    };
  },
}));

it(`renders correctly`, () => {
  const tree = renderer.create(<Reference />);
  expect(tree).toMatchSnapshot();
});
