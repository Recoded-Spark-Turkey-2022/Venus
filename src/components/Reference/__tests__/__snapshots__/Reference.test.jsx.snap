// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col md:flex-row justify-around items-center py-12 "
  style={
    Object {
      "opacity": 0,
      "transform": "scale(0.5) translateZ(0)",
    }
  }
>
  <div
    className="flex justify-center items-center "
  >
    <img
      alt="World Map"
      className="w-2/3 md:w-full"
      src="world_map_ 1.svg"
    />
  </div>
  <div
    className="flex justify-end w-full md:w-1/2 "
  >
    <Swiper
      autoplay={
        Object {
          "delay": 2500,
          "disableOnInteraction": false,
        }
      }
      className="mySwiper"
      modules={
        Array [
          "Pagination",
          undefined,
        ]
      }
      pagination={
        Object {
          "clickable": true,
          "dynamicBullets": true,
        }
      }
    >
      <SwiperSlide>
        <div
          className="w-1/2 reference-div text-center sm:text-left"
        >
          <h3
            className="pt-2"
          >
            Two lines of a short testimonial from someone want to say something, and can say more to explain.
          </h3>
          <p
            className="text-darkGrey pt-2"
          >
            Louis Li
          </p>
          <p
            className="text-darkGrey text-sm"
          >
            Trainer at Recoded
          </p>
        </div>
      </SwiperSlide>
      <SwiperSlide>
        <div
          className="w-1/2 reference-div text-center sm:text-left"
        >
          <h3
            className="pt-2"
          >
            Two lines of a short testimonial from someone want to say something, and can say more to explain.
          </h3>
          <p
            className="text-darkGrey pt-2"
          >
            Louis Li
          </p>
          <p
            className="text-darkGrey text-sm"
          >
            Trainer at Recoded
          </p>
        </div>
      </SwiperSlide>
      <SwiperSlide>
        <div
          className="w-1/2 reference-div text-center sm:text-left"
        >
          <h3
            className="pt-2"
          >
            Two lines of a short testimonial from someone want to say something, and can say more to explain.
          </h3>
          <p
            className="text-darkGrey pt-2"
          >
            Louis Li
          </p>
          <p
            className="text-darkGrey text-sm"
          >
            Trainer at Recoded
          </p>
        </div>
      </SwiperSlide>
    </Swiper>
  </div>
</div>
`;
